### DISCUSSION TITLE
Make the UI HACS compatible

### DISCUSSION THREAD

**Discussion by CM000n** [UI-Lovelace-Minimalist/UI#123] [Ideas]
I recently saw that now some custom themes can already be installed via HACS. (For example Dwains Dahrboard)
Of course, these currently offer more logic, in the setup than our UI but maybe we also have a way to make our UI available via HACS?
This would of course be the most comfortable for the less technically savvy user.
Unfortunately, I'm not quite clear yet what needs to be done to make this happen. Any support would be as very welcome :-)

**Comment by stokkie90** [comment]
Will see if i can make something work. But it will be after the holidays.

**Comment by CM000n** [comment]
I have just posted the call for a logo creation in the Home Assistant forum. I think this is a very cool action that several people could take part in: 

https://community.home-assistant.io/t/lovelace-ui-minimalist/322687/889
